webui/react/src/components/Markdown.tsx

```diff
@@ -92,7 +92,7 @@ const Markdown: React.FC<Props> = ({
   return (
     <div aria-label="markdown-editor" className={css.base} tabIndex={0}>
       {editing && !disabled ? (
-        <Tabs className="no-padding" items={tabItems} />
+        <Pivot items={tabItems} />
```

Comment:
Removing "no-padding" does not seem off in this case, so no additional changes were applied
<img width="855" alt="Screen Shot 2023-01-10 at 10 56 55 AM" src="https://user-images.githubusercontent.com/40620519/211614456-98e97137-7974-4bf7-a358-a30ea4027b3e.png">